doc/conf.py

```diff
@@ -1,7 +1,11 @@
 from documenteer.conf.pipelinespkg import *
 
 
+<<<<<<< HEAD
 project = "meas_algorithms"
+=======
+project = "example"
+>>>>>>> a3987573 (Update config file following updated build)
```

Comment:
Please revert the change to this file entirely. Looks like it's on a commit by itself, so you can do an interactive rebase and put `d` to drop that commit.